model/follow_up_v2.go

```diff
@@ -0,0 +1,55 @@
+package model
+
+import "github.com/incident-io/singer-tap/client"
+
+type followUpV2 struct{}
+
+var FollowUpV2 followUpV2
+
+func (followUpV2) Schema() Property {
+	return Property{
+		Types: []string{"object"},
+		Properties: map[string]Property{
+			"id": {
+				Types: []string{"string"},
+			},
+			"incident_id": {
+				Types: []string{"string"},
+			},
+			"priority": Optional(FollowUpPriorityV2.Schema()),
+			"status": {
+				Types: []string{"string"},
+			},
+			"title": {
+				Types: []string{"string"},
+			},
+			"description": {
+				Types: []string{"string", "null"},
+			},
+			"external_issue_reference": Optional(ExternalIssueReferenceV2.Schema()),
+			"completed_at":             DateTime.Schema(),
+			"created_at":               DateTime.Schema(),
+			"updated_at":               DateTime.Schema(),
+		},
```

Comment:
Do we not return the `owner` here?

Reply:
Oh you're right - we should have the assignee here.